pkg/imageverifycache/client.go

```diff
@@ -0,0 +1,116 @@
+package imageverifycache
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+)
+
+type cache struct {
+	logger         logr.Logger
+	isCacheEnabled bool
+	maxSize        int64
+	TTL            int64
```

Comment:
Do we need to expose `ttl`?

Reply:
[We should allow user to specify the max duration for a cache](https://docs.google.com/document/d/1L_JVyDEVdZZPdLaFjsv7_KfbJ2As3Nzt48Pu2xqgr_U/edit?disco=AAAA1TqihLQ), that why I am exposing it. I am only allowing change in that value when the cache is initially created

Reply:
How about exposing a Get/Set interface?